examples/smart-accounts/signers/dynamic/src/components/TransactionForm.tsx

```diff
@@ -62,7 +64,7 @@ export default function TransactionForm() {
       <div className="mt-8">
         <SendUserOperationButton
           to={address as Address}
-          value={parseEther(value)}
+          value={isValidValue ? parseEther(value) : 0n}
```

Comment:
nit: this could be broken into a variable to avoid complicating the react component